metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -280,9 +286,11 @@ def get_schema_metadata(
     return schema_metadata
 
 
-class SQLAlchemySource(Source):
+class SQLAlchemySource(SampleableSource):
     """A Base class for all SQL Sources that use SQLAlchemy to extend"""
 
+    SAMPLE_SIZE: int = 50
```

Comment:
do we want this to be tuneable?

Reply:
Yes, although this field is no longer used, will remove. Now, the entire query is passed in via the k8s recipe file (including the `LIMIT`)